src/daemon/inout_route/chat.rs

```diff
@@ -92,15 +97,99 @@ pub fn get_chat(ctx: &DaemonContext, neigh: Fingerprint) -> Vec<(bool, String, S
         .collect()
 }
 
-pub async fn send_chat_msg(ctx: &DaemonContext, dest: Fingerprint, msg: String) {
+pub async fn send_chat_msg(
+    ctx: &DaemonContext,
+    dest: Fingerprint,
+    msg: String,
+) -> anyhow::Result<()> {
     let chats = ctx.get(CHATS);
+    let my_sk = *ctx.get(GLOBAL_IDENTITY);
+    let settlements = ctx.get(SETTLEMENTS);
 
-    if let Some(client) = chats.clients.get(&dest) {
+    if msg.starts_with("!settle ") {
```

Comment:
Maybe we can use an enum with a custom impl for `FromStr` instead of hardcoding the strings?